openedxscorm/scormxblock.py

```diff
@@ -15,15 +15,22 @@
 from django.utils import timezone
 from django.utils.module_loading import import_string
 from webob import Response
-import importlib_resources
 from six import string_types
-
 from web_fragments.fragment import Fragment
 from xblock.core import XBlock
 from xblock.completable import CompletableXBlockMixin
 from xblock.exceptions import JsonHandlerError
 from xblock.fields import Scope, String, Float, Boolean, Dict, DateTime, Integer
 
+try:
+    # Older Open edX releases (Quince and earlier) install a backported version of
+    # importlib.resources: https://pypi.org/project/importlib-resources/
+    import importlib_resources
```

Comment:
I don't understand why this would be a problem in Redwood. The following commands are successful in tutor v18.0.0:

```
tutor local run --no-deps lms python -c "import importlib_resources"
tutor dev run --no-deps lms python -c "import importlib_resources"
```

Reply:
@regisb Whoops, you're right. Git blame tells me that [this change happened after the Redwood cut](https://github.com/openedx/edx-platform/pull/34788), so it's only affecting Nightly currently. I'll update the comments to say that it's a Redwood->Sumac difference rather than Quince->Redwood.

Reply:
Done